terraform/addons/mdm/outputs.tf

```diff
@@ -0,0 +1,31 @@
+output "extra_environment_variables" {
+  value = {
+    FLEET_MDM_APPLE_ENABLE         = "1"
+    FLEET_DEV_MDM_ENABLED          = "1"
+    FLEET_MDM_APPLE_SERVER_ADDRESS = var.public_domain_name
+  }
+}
+
+output "extra_secrets" {
+  value = {
+    FLEET_MDM_APPLE_SCEP_CERT_BYTES       = "${aws_secretsmanager_secret.scep.arn}:crt::"
```

Comment:
Do we care long-term that these are easily visible in the task definition yaml like this?  I figure probably not since if you can see that, you can also probably open the secret itself with our access controls.  I'll approve it for now regardless.

Reply:
no, the actual values are secret. this is just a key in a data structure not the value.